### PR TITLE
added major cybersecurity companies from Nepal

### PR DESCRIPTION
**Pull request by V35HR4J**
Added major cybersecurity companies from Nepal :)

### CODE REVIEW

File: data/companies.json [hunk 1]
```diff
@@ -13,6 +13,34 @@
...
+      "github": "https://github.com/cryptogennepal",
+      "instagram": "https://instagram.com/CryptoGenNepal",
+      "linkedin": "https://np.linkedin.com/company/cryptogennepal",
+      "twitter": "https://twitter.com/CryptoGenNepal",
+      "youtube": "https://www.youtube.com/channel/UC2SCx0gZ--DJNI6vXrSpzdw"
+    }
+  },
+  {
+    "name": "Cynical Technology",
+    "establishedAt": "null",
```
Comment (thenishantsapkota) [inline data/companies.json:32]: Why is `establishedAt` null here?

File: data/companies.json [hunk 2]
```diff
@@ -13,6 +13,34 @@
...
+  {
+    "name": "CryptoGen Nepal",
+    "establishedAt": 1553609490,
+    "location": "Naaxl, Kathmandu",
+    "website": "https://cryptogennepal.com/",
+    "socials": {
+      "facebook": "https://www.facebook.com/CryptoGenNepal",
+      "github": "https://github.com/cryptogennepal",
+      "instagram": "https://instagram.com/CryptoGenNepal",
+      "linkedin": "https://np.linkedin.com/company/cryptogennepal",
```
Comment (thenishantsapkota) [inline data/companies.json:25]: Can we have the `https://linkedin.com` domain?
We are trying to maintain standards

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Please change all LinkedIn urls to the standard one.
`https://linkedin.com/...`

**Review by TheLearneer — COMMENTED** [review]
First of all, thank you for your contributions @V35HR4J 🙏 

On top of what @thenishantsapkota just mentioned, I would love to know the source for the established date.
I quickly looked into the website and couldn't find much of those information 😅

File: data/companies.json (hunk 2 above)
Comment (V35HR4J) [reply data/companies.json]: Done

**Review by V35HR4J — COMMENTED** [review]
finished

**Comment by V35HR4J** [comment]
> First of all, thank you for your contributions @V35HR4J 🙏
> 
> On top of what @thenishantsapkota just mentioned, I would love to know the source for the established date. I quickly looked into the website and couldn't find much of those information 😅

Hi @TheLearneer 
you can find the sources here respectively:
https://csrinepal.org/team/cryptogen-nepal-pvt-ltd/
https://www.crunchbase.com/organization/eminence-ways
https://www.crunchbase.com/organization/vairav-technology

**Comment by thenishantsapkota** [comment]
> finished



> finished

Could you please check the comment of @TheLearneer?
Thank you :pray:

**Review by TheLearneer — APPROVED** [review]
LGTM, thank you for your contribution @V35HR4J 🙇

**Review by thenishantsapkota — APPROVED** [review]
LGTM!
Thank you for your contributions @V35HR4J 
Join our discord server to claim your special Contributors role.
Cheers!

**Pull request merged**